parser.y

```diff
@@ -203,15 +203,12 @@ rule
 
   token_declaration: id int_opt alias { result = val }
 
-  rule_declaration: "%rule" IDENTIFIER "(" rule_args ")" ":" rule_rhs_list
+  rule_declaration: "%rule" IDENTIFIER "(" parameterizing_args ")" ":" rule_rhs_list
```

Comment:
`rule_args` is a list of `IDENTIFIER`. On the other hand `parameterizing_args` is a list of `symbol`. `symbol` means `IDENTIFIER`, `CHARACTER` or `STRING`.
In my understanding, using `symbol` for arguments make sense, but only `IDENTIFIER` makes sense for parameters.

Reply:
Ah. Sorry. I was confused 🙇‍♂ You are right. Thank you.

Reply:
No worries!